pkg/config/config.go

```diff
@@ -155,6 +155,14 @@ func init() {
 	Datadog.SetDefault("use_v2_api.series", false)
 	Datadog.SetDefault("use_v2_api.events", false)
 	Datadog.SetDefault("use_v2_api.service_checks", false)
+	// Serializer: allow user to blacklist any kind of payload to be sent
+	BindEnvAndSetDefault("serializer_enable_event", true)
+	BindEnvAndSetDefault("serializer_enable_series", true)
+	BindEnvAndSetDefault("serializer_enable_service_check", true)
+	BindEnvAndSetDefault("serializer_enable_sketches", true)
+	BindEnvAndSetDefault("serializer_enable_metadata", true)
```

Comment:
Even though it's good to also have a `metadata` option for consistency, in this case I think we should remove it and add a comment here to explain why it's not implemented:

- disabling sending all metadata can have far-reaching consequences (e.g. billing) that we (including the support team) may not foresee. If there's a use case for it, I want to be sure the dev team is pinged before the option is even tried.
- in all known cases, the `enable_metadata_collection` option should be preferred as it's more selective and already well known and documented.